src/components/ConnectedNodes.test.tsx

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-restricted-globals */
```

Comment:
Let's avoid this.
I think you should look into `src/types/globals.d.ts`, to add `speechSynthesis` to the Window global